### PR TITLE
Cancellation Flow: Tweak Cancellation Reasons

### PR DESCRIPTION
**Pull request by taggon**
#### Changes proposed in this Pull Request

* As a part of the solution-oriented cancellation flow project pcbrnV-3O0-p2,  this PR tweaks the cancellation reasons we show in the cancellation flow, so they are categorized into main/subgroups.

#### Testing instructions

* Try to cancel the subscription of your paid plan.
  Cancel_Purchase_‹_Premium_Monthly_—_WordPress_com
* On the "Share your feedback" page, you will see the new cancellation reason options in the two selectboxes. The full list of the options can be found on pcbrnV-3O0-p2.
  Cancel_Purchase_‹_Premium_Monthly_—_WordPress_com
* In addition to the two select boxes, there should be some upsell nudges for specific cases.
  - Lower plan than Buisness Plan, "Can’t use a plugin/theme" => Upgrade to atomic upsell
  - Business plan, "Can’t use a plugin/theme" => Upgrade to business atomic upsell
  - Premium Plan, "Too expensive" or "I want a cheaper plan" => Downgrade to Personal plan upsell
  - An yearly/bi-yearly plan, "Too expensive" or "I want a cheaper plan" => Downgrade to a corresponding monthly plan
  - A monthly plan on a site that has never consumed the free offer, "I don’t have time." or "My site is not ready" => Free month offer

### CODE REVIEW

File: client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+export const CANCELLATION_REASONS: CancellationReason[] = [
+	{
+		value: 'price/budget',
+		label: translate( 'Price/Budget' ),
+		selectOptions: [
+			PLACEHOLDER,
+			{
+				value: 'tooExpensice',
+				label: translate( 'Too expensive' ),
```
Comment (a8ci18n) [inline client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'It's too expensive.' )` *ES Score: 9*

File: client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+				value: 'noTime',
+				label: translate( 'I don’t have time.' ),
+			},
+			{
+				value: 'needProfessionalHelp',
+				label: translate( 'Need professional help to build my site' ),
+			},
+			{
+				value: 'siteIsNotReady',
+				label: translate( 'My site is not ready' ),
```
Comment (a8ci18n) [inline client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'My site is not ready.' )` *ES Score: 9*

File: client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+		label: translate( 'Missing features' ),
+		selectOptions: [
+			PLACEHOLDER,
+			{
+				value: 'otherFeatures',
+				label: translate( 'Other features' ),
+			},
+			{
+				value: 'eCommerceFeatures',
+				label: translate( 'eCommerce features' ),
```
Comment (a8ci18n) [inline client/components/marketing-survey/cancel-purchase-form/cancellation-reasons.ts:121]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Advanced eCommerce Features' )` *ES Score: 7*

**Review by southp — APPROVED** [review]
Confirmed that it works as described. LGTM and thanks for making the code a lot more organized 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7143744

Thank you @taggon for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7143744).

**Pull request merged**